src/api/bucket_api.js

```diff
@@ -1373,7 +1373,8 @@ module.exports = {
             required: ['resource'],
             properties: {
                 resource: { type: 'string' },
-                path: { type: 'string' }
+                path: { type: 'string' },
+                force_md5_etag: { type: 'boolean' }
```

Comment:
`force_md5_etag` should be added to get\set methods , and not `namespace_resource_config`. e.g.: `create_bucket`, `update_bucket_params`, `bucket_info`